### DISCUSSION TITLE
How can I communicate with vision models via api?

### DISCUSSION THREAD

**Discussion by mh292929** [xtekky/gpt4free#2510] [Q&A]
hi. How can I communicate with vision models via api?

**Comment by Juxoola** [comment]
https://github.com/xtekky/gpt4free/blob/main/docs/client.md#using-a-vision-model
https://github.com/xtekky/gpt4free/blob/main/docs/async_client.md#using-a-vision-model

**Comment by hlohaus** [comment]
Also there is this guide:
https://github.com/xtekky/gpt4free/blob/main/docs/vision.md

@MH140000
